src/select/index.js

```diff
@@ -29,7 +29,7 @@ export {
   StyledIconsContainer,
   StyledSelectArrow,
   StyledClearIcon,
-  StyledSearchIcon,
+  StyledSearchIconContainer,
```

Comment:
We should keep the `StyledSearchIcon` export that equals `StyledSearchIconContainer`, to keep it as non-breaking.

Reply:
Good catch. Updated!